transports/msmq/index.md

```diff
@@ -104,9 +103,9 @@ MSMQ permissions are defined in the [MessageQueueAccessRights](https://msdn.micr
 
 NOTE: To increase security and further lock down MSMQ send/receive permissions remove `Everyone` and `Anonymous` and grant specific permissions to the subset of accounts that need them.
 
-NOTE: From Version 6, if the default queue permissions are set, a log message will be written during the transport startup, reminding that the queue has default permissions. During development, if running with an attached debugger, this message will be logged as `INFO` level, otherwise `WARN`.
+NOTE: From Versions 6 and above, if the default queue permissions are set, a log message will be written during the endpoint startup, reminding that the queue has default permissions and might require stricter permissions for production. During development, if running with an attached debugger, this message will be logged as `INFO` level, otherwise `WARN`.
```

Comment:
"From" doesn't flow well with "Versions X and above". Maybe just "In Versions 6 and above"?